generators/app/prompts.js

```diff
@@ -125,3 +128,75 @@ function askForTestOpts() {
         done();
     }.bind(this));
 }
+
+function httpGet(url, onSuccess, onFail) {
+    http.get(url, function(res) {
+        var body = '';
+        res.on('data', function(chunk) {
+            body += chunk;
+        });
+        res.on('end', function() {
+            onSuccess(body);
+        });
+    }).on('error', onFail);
+}
+
+function askModulesToBeInstalled(done, generator) {
+    httpGet('http://npmsearch.com/query?fields=name,keywords,description,author,homepage,version,repository,created&q=keywords:jhipster-module&start=0&size=50',
+        function(body) {
+            var moduleResponse = JSON.parse(body);
+            var choices = [];
+            moduleResponse.results.forEach(function (modDef) {
+                choices.push({
+                    value: { name:modDef.name, version:modDef.version},
+                    name: '(' + modDef.name + '-' + modDef.version + ') '+ modDef.description + ' [' + modDef.author + ']'
+                });
+            });
+            if (choices.length > 0) {
+                generator.prompt({
+                    type: 'checkbox',
+                    name: 'otherModules',
+                    message: 'Which other modules would you like to use?',
+                    choices: choices,
+                    default: []
+                }).then(function (prompt) {
+                    // [ {name: [moduleName], version:[version]}, ...]
+                    generator.otherModules = [];
+                    prompt.otherModules.forEach(function(module) {
+                        generator.otherModules.push({name:module.name[0], version:module.version[0]});
+                    });
+                    done();
+                }.bind(generator));
+            } else {
+                done();
+            }
+        },
+        function (error) {
+            this.warning('Unable to contact server for modules: '+error.message);
```

Comment:
can we change to `Unable to contact server to fetch additional modules: ${error.message}` using backtick notation